js/src/apbct-public.js

```diff
@@ -247,14 +247,19 @@
 					(form.classList && form.classList.contains('give-form')) || // GiveWP
 					(form.id && form.id === 'ult-forgot-password-form') || //ult forgot password
 					(form.id && form.id.toString().indexOf('calculatedfields') !== -1) // CalculatedFieldsForm
-				)
+				) {
 					continue;
-
```

Comment:
I don't really understand why do we need to delete this code? Are there some problems with it?

If so, please, don't forget to delete the code on 234 line.

Reply:
Fix: Code. Unused variables removed.   https://github.com/CleanTalk/wordpress-antispam/commit/5a27d797cb86856bd2235472928bae188f0d3abb